autoload/neoterm/term/neovim.vim

```diff
@@ -5,7 +5,7 @@ endfunction
 let s:neovim = {}
 
 function! s:neovim.new(opts) abort
-  if g:neoterm_direct_open_repl
+  if g:neoterm_direct_open_repl && len(g:neoterm_repl_command) < 2
```

Comment:
What if ` g:neoterm_direct_open_repl` is set and `len(g:neoterm_repl_command)` is `>= 2`?

Reply:
This opens an empty terminal, as I say in my first post it's a dirty bugfix.
I can't find a solution to pass lists to termopen()